### ISSUE TITLE
The address property in the userinfo endpoint should be an object instead of a string

### ISSUE THREAD

**Issue by sjillen** [ConsumerDataRight/mock-data-recipient#67]
**Describe the bug**
When calling the userinfo endpoint with the address scope, the mock-data-recipient expects a string for the address property while the openID connect specs expect a JSON object.
References:
- [section 5.1 Standard Claims](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo)
- [CDS Scopes and Claims requirements](https://consumerdatastandardsaustralia.github.io/standards/#scopes-and-claims)

**To Reproduce**
1.  Register a data_holder and a user with its address parsed as an object
2.  Call userinfo endpoint with the `address` scope
3.  The mock data recipient returns a 500 error

**Expected behaviour**
The userinfo endpoint is supposed to return the userinfo data with the address as a JSON object instead of a string as per the openID specs. The address JSON object contains the following properties:
```
"address": {
    "formatted": "...",
    "street_address": "...",
    "locality": "...",
    "region": "...",
    "postal_code": "...",
    "country": "..."
}
```

**Comment by CDR-AndrewG** [comment]
Hi @sjillen,

Thank you for reaching out and raising this issue. We have logged the issue and will include a fix in a future release.

**Comment by CDR-HansenS** [comment]
Hi @sjillen,

This issue has now been fixed in the Mock Data Recipient v1.2.5 from our recent production release.
